.github/workflows/dashboards-ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Dashboards CI
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+    - 'dashboards/**'
+    - 'packaging/obs/grafana-ha-cluster-dashboards/**'
+    - '.github/workflows/dashboards*'
+  pull_request:
+    branches: [ master ]
+    paths:
+    - 'dashboards/**'
+    - 'packaging/obs/grafana-ha-cluster-dashboards/**'
+    - '.github/workflows/dashboards*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+    - run: npm -g install jsonlint
+    - name: validate JSON
+      run: find dashboards -name "*.json" -type f -exec jsonlint -c {} \;
+
+  obs-commit:
+    needs: build
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    container: shap/continuous_deliver
```

Comment:
is this a container we have ? maybe we need to update confluence page regarding this

Reply:
yeah this is hosted on docker hub and sourced from https://github.com/arbulu89/continuous-delivery

Reply:
ok I think this was an `howto` tutorial. I will move it to `CI` session. I think personally the howto section is somehow to much generic like `utils` directory without any context

Reply:
@stefanotorresi  i'm moving the page on the CI section. feel free to add it maybe with github action context too

Reply:
yeah, good call!